app/app.go

```diff
@@ -82,13 +82,13 @@ import (
 	upgradeclient "github.com/cosmos/cosmos-sdk/x/upgrade/client"
 	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	appparams "github.com/crypto-org-chain/chain-main/app/params"
-	"github.com/crypto-org-chain/chain-main/x/chainmain"
-	chainmainkeeper "github.com/crypto-org-chain/chain-main/x/chainmain/keeper"
-	chainmaintypes "github.com/crypto-org-chain/chain-main/x/chainmain/types"
-	supply "github.com/crypto-org-chain/chain-main/x/supply"
-	supplykeeper "github.com/crypto-org-chain/chain-main/x/supply/keeper"
-	supplytypes "github.com/crypto-org-chain/chain-main/x/supply/types"
+	appparams "github.com/crypto-org-chain/chain-main/v1/app/params"
```

Comment:
What's the version here used for?

Reply:
that's to signal the global version / start the mainnet versioning -- afaik I guess for v1, it's optional, but we can start. See Cosmos Hub: https://github.com/cosmos/gaia/commit/2bb04266266586468271c4ab322367acbf41188f

Reply:
why cosmos-sdk don't use this kind of versioning?

Reply:
Is it only for documenting purpose? I think it'll always build the version of code in current repo, no matter what the version number is?

Reply:
just for protocol level, use versions.
in coding level, use just use it as configured as current
otherwise, it would be too complex in development.

Reply:
@yihuang AFAIK Cosmos SDK is "pre-1.0", so they don't guarantee anything / they don't want to signal any long-term maintenance at this point. 

@leejw51crypto it's more for the future -- https://blog.golang.org/v2-go-modules